stockflux-securities-master/src/services/SecuritiesService.js

```diff
@@ -0,0 +1,8 @@
+export async function getSecuritiesData() {
+  //temporarily using local host, will need to change to the AWS location
+  const response = await fetch(
+    "https://g3amedujs2.execute-api.eu-west-2.amazonaws.com/dev/api/securities-v2"
```

Comment:
Can we look at using the custom data to get the base URL of the API, rather than hard-coding it? You might need to do a bit of configuration locally to get it to use the domain you want. Take a look at the `StockfluxService` that the other guys wrote for details on how to do this.

Reply:
Okay this has been changed and I didn't commit the app.dev.json file change which will correspond to this.